diff --git a/composer.json b/composer.json
index 353cc21..648e79f 100644
--- a/composer.json
+++ b/composer.json
@@ -35,10 +35,6 @@
         },
         {
             "type": "git",
-            "url": "https://github.com/alnutile/MinkExtension.git"
-        },
-        {
-            "type": "git",
             "url": "http://git.drupal.org/project/drupalextension.git"
         }
     ],
diff --git a/src/Drupal/BehatEditor/BehatEditorBatchTypeTag.php b/src/Drupal/BehatEditor/BehatEditorBatchTypeTag.php
index 28bc6b4..6b9653c 100644
--- a/src/Drupal/BehatEditor/BehatEditorBatchTypeTag.php
+++ b/src/Drupal/BehatEditor/BehatEditorBatchTypeTag.php
@@ -67,7 +67,7 @@ class BehatEditorBatchTypeTag extends  BehatEditorBatchType {
     }
 
     protected function wrapUp(&$fields) {
-        file_unmanaged_delete_recursive(file_build_uri("/behat_batch/{$this->rid}"));
+        //file_unmanaged_delete_recursive(file_build_uri("/behat_batch/{$this->rid}"));
         parent::wrapUp($fields);
     }
 
diff --git a/src/Drupal/BehatEditor/BehatEditorRun.php b/src/Drupal/BehatEditor/BehatEditorRun.php
index 2b1950a..4981a9a 100644
--- a/src/Drupal/BehatEditor/BehatEditorRun.php
+++ b/src/Drupal/BehatEditor/BehatEditorRun.php
@@ -130,7 +130,6 @@ class BehatEditorRun {
         $command = self::behatCommandArray();
 
         //@todo move this into a shared method for exec and execDrush
-        $this->settings['context'] = $context1;
         $behat_yml_path = new GenerateBehatYml($this->settings);
         $this->behat_yml = $behat_yml_path->writeBehatYmlFile();
 
@@ -160,7 +159,7 @@ class BehatEditorRun {
      *   if the user is running selenium
      * @return array
      */
-    public function execDrush($javascript = FALSE, $tag_include = FALSE, $profile = 'default', $settings = array(),  $context1 = 'behat_run') {
+    public function execDrush($javascript = FALSE, $tag_include = FALSE, $profile = 'default', $settings = array()) {
         if($javascript == TRUE) {
             $tags_exclude = '';
         } else {
@@ -174,7 +173,7 @@ class BehatEditorRun {
         }
 
         $this->tags = "$tag_include $tags_exclude";
-        $this->settings['context'] = $context1;
+        $this->settings = $settings;
 
         $command = self::behatCommandArray();
 
