diff --git a/behat_editor.install b/behat_editor.install
index 86df2b7..573d090 100644
--- a/behat_editor.install
+++ b/behat_editor.install
@@ -53,7 +53,7 @@ function behat_editor_requirements($phase) {
     return $requirements;
 }
 
-function behat_editor_enable(){
+function behat_editor_install(){
     $tables = behat_editor_schema();
     foreach($tables as $key => $value) {
         if(!db_table_exists($key)) {
diff --git a/js/behat_editor_index.js b/js/behat_editor_index.js
index 0dfe109..acf7453 100644
--- a/js/behat_editor_index.js
+++ b/js/behat_editor_index.js
@@ -12,9 +12,6 @@
                 e.preventDefault();
                 var rid = $(this).data('rid');
                 var body = Drupal.settings.behat_editor.results[rid].results;
-                if (body instanceof Array) {
-                    body = body.join("<br />");
-                }
                 $('#modalResults div.test').html(body);
                 $('#modalResults').modal();
             });
diff --git a/js/behat_editor_results.js b/js/behat_editor_results.js
index ce58ef3..9f9acff 100644
--- a/js/behat_editor_results.js
+++ b/js/behat_editor_results.js
@@ -9,9 +9,6 @@
             e.preventDefault();
             var rid = $(this).attr('id');
             var body = $('body').data('behat_results')[rid];
-            if (body instanceof Array) {
-                body = body.join("<br />");
-            }
             body = body.replace(/,/g, "<br />");
             $('#modalResults div.test').html(body);
             $('#modalResults').modal();
